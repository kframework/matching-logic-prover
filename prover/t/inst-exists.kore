axiom f5: \functionalPattern(5)

claim \exists{I{Int}} gte(I{Int}, 3)
strategy inst-exists(I{Int}, 5, apply(f5, fail))
       . smt-cvc4
