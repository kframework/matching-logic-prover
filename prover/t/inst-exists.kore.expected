<prover>
  <exit-code>
    0
  </exit-code>
  <goals>
    <goal>
      id: ., parent: .
      <claim>
        \and ( .Patterns )
      </claim>
      <k>
        .
      </k>n<expected>
        .
      </expected>
      <local-context>
        .LocalDeclCellSet
      </local-context>
    <trace>
      .
    </trace>
  </goals>
  <declarations>
    <declaration>
      axiom ax0 : \exists { I { Int } , .Patterns } gte ( I { Int } , 3 , .Patterns )
    </declaration> <declaration>
      axiom f5 : \functionalPattern ( 5 )
    </declaration>
  </declarations>
</prover>
